[
    {
        "url": "https://api.github.com/repos/microsoft/onnxruntime/issues/comments/1192727611",
        "html_url": "https://github.com/microsoft/onnxruntime/issues/12290#issuecomment-1192727611",
        "issue_url": "https://api.github.com/repos/microsoft/onnxruntime/issues/12290",
        "id": 1192727611,
        "node_id": "IC_kwDOCVq1mM5HF5Q7",
        "user": {
            "login": "baijumeswani",
            "id": 12852605,
            "node_id": "MDQ6VXNlcjEyODUyNjA1",
            "avatar_url": "https://avatars.githubusercontent.com/u/12852605?v=4",
            "gravatar_id": "",
            "url": "https://api.github.com/users/baijumeswani",
            "html_url": "https://github.com/baijumeswani",
            "followers_url": "https://api.github.com/users/baijumeswani/followers",
            "following_url": "https://api.github.com/users/baijumeswani/following{/other_user}",
            "gists_url": "https://api.github.com/users/baijumeswani/gists{/gist_id}",
            "starred_url": "https://api.github.com/users/baijumeswani/starred{/owner}{/repo}",
            "subscriptions_url": "https://api.github.com/users/baijumeswani/subscriptions",
            "organizations_url": "https://api.github.com/users/baijumeswani/orgs",
            "repos_url": "https://api.github.com/users/baijumeswani/repos",
            "events_url": "https://api.github.com/users/baijumeswani/events{/privacy}",
            "received_events_url": "https://api.github.com/users/baijumeswani/received_events",
            "type": "User",
            "site_admin": false
        },
        "created_at": "2022-07-22T16:08:12Z",
        "updated_at": "2022-07-22T16:08:12Z",
        "author_association": "MEMBER",
        "body": "This should not happen for rocm version of onnxruntime. Is it possible that the package is a cuda package and not a rocm one?\r\nAre you able to run this command:\r\n```py\r\npython -c \"from onnxruntime.capi import build_and_package_info as ort_info; print(ort_info.rocm_version)\"\r\n```\r\nAlso try to run\r\n```py\r\npython -c \"from onnxruntime.capi import build_and_package_info as ort_info; print(ort_info.cuda_version)\"\r\n```",
        "reactions": {
            "url": "https://api.github.com/repos/microsoft/onnxruntime/issues/comments/1192727611/reactions",
            "total_count": 0,
            "+1": 0,
            "-1": 0,
            "laugh": 0,
            "hooray": 0,
            "confused": 0,
            "heart": 0,
            "rocket": 0,
            "eyes": 0
        },
        "performed_via_github_app": null
    },
    {
        "url": "https://api.github.com/repos/microsoft/onnxruntime/issues/comments/1192733829",
        "html_url": "https://github.com/microsoft/onnxruntime/issues/12290#issuecomment-1192733829",
        "issue_url": "https://api.github.com/repos/microsoft/onnxruntime/issues/12290",
        "id": 1192733829,
        "node_id": "IC_kwDOCVq1mM5HF6yF",
        "user": {
            "login": "rui-ren",
            "id": 15321482,
            "node_id": "MDQ6VXNlcjE1MzIxNDgy",
            "avatar_url": "https://avatars.githubusercontent.com/u/15321482?v=4",
            "gravatar_id": "",
            "url": "https://api.github.com/users/rui-ren",
            "html_url": "https://github.com/rui-ren",
            "followers_url": "https://api.github.com/users/rui-ren/followers",
            "following_url": "https://api.github.com/users/rui-ren/following{/other_user}",
            "gists_url": "https://api.github.com/users/rui-ren/gists{/gist_id}",
            "starred_url": "https://api.github.com/users/rui-ren/starred{/owner}{/repo}",
            "subscriptions_url": "https://api.github.com/users/rui-ren/subscriptions",
            "organizations_url": "https://api.github.com/users/rui-ren/orgs",
            "repos_url": "https://api.github.com/users/rui-ren/repos",
            "events_url": "https://api.github.com/users/rui-ren/events{/privacy}",
            "received_events_url": "https://api.github.com/users/rui-ren/received_events",
            "type": "User",
            "site_admin": false
        },
        "created_at": "2022-07-22T16:15:48Z",
        "updated_at": "2022-07-22T16:18:39Z",
        "author_association": "MEMBER",
        "body": "Hi @baijumeswani, thanks for your quick reply,\r\n\r\nI installed onnxruntime using \r\n```\r\npython -m pip install --pre onnxruntime-training -f https://download.onnxruntime.ai/onnxruntime_nightly_rocm52.html\r\n```\r\n\r\n- ran\r\n```\r\npython -c \"from onnxruntime.capi import build_and_package_info as ort_info; print(ort_info.rocm_version)\"\r\n```\r\nI got error of `RuntimeError: module compiled against API version 0xe but this version of numpy is 0xd`\r\n\r\n- ran\r\n```\r\npython -c \"from onnxruntime.capi import build_and_package_info as ort_info; print(ort_info.cuda_version)\"\r\n```\r\nI got the error of \r\n```\r\nRuntimeError: module compiled against API version 0xe but this version of numpy is 0xd\r\n10.2\r\n```\r\n\r\n",
        "reactions": {
            "url": "https://api.github.com/repos/microsoft/onnxruntime/issues/comments/1192733829/reactions",
            "total_count": 0,
            "+1": 0,
            "-1": 0,
            "laugh": 0,
            "hooray": 0,
            "confused": 0,
            "heart": 0,
            "rocket": 0,
            "eyes": 0
        },
        "performed_via_github_app": null
    },
    {
        "url": "https://api.github.com/repos/microsoft/onnxruntime/issues/comments/1192849939",
        "html_url": "https://github.com/microsoft/onnxruntime/issues/12290#issuecomment-1192849939",
        "issue_url": "https://api.github.com/repos/microsoft/onnxruntime/issues/12290",
        "id": 1192849939,
        "node_id": "IC_kwDOCVq1mM5HGXIT",
        "user": {
            "login": "ytaous",
            "id": 4484531,
            "node_id": "MDQ6VXNlcjQ0ODQ1MzE=",
            "avatar_url": "https://avatars.githubusercontent.com/u/4484531?v=4",
            "gravatar_id": "",
            "url": "https://api.github.com/users/ytaous",
            "html_url": "https://github.com/ytaous",
            "followers_url": "https://api.github.com/users/ytaous/followers",
            "following_url": "https://api.github.com/users/ytaous/following{/other_user}",
            "gists_url": "https://api.github.com/users/ytaous/gists{/gist_id}",
            "starred_url": "https://api.github.com/users/ytaous/starred{/owner}{/repo}",
            "subscriptions_url": "https://api.github.com/users/ytaous/subscriptions",
            "organizations_url": "https://api.github.com/users/ytaous/orgs",
            "repos_url": "https://api.github.com/users/ytaous/repos",
            "events_url": "https://api.github.com/users/ytaous/events{/privacy}",
            "received_events_url": "https://api.github.com/users/ytaous/received_events",
            "type": "User",
            "site_admin": false
        },
        "created_at": "2022-07-22T18:51:53Z",
        "updated_at": "2022-07-22T18:51:53Z",
        "author_association": "MEMBER",
        "body": "can u check your numpy version? upgrade it and retry? \r\nWhich docker image you are using? platform one? can u also try \"ptebic.azurecr.io/internal/azureml/aifx/stable-ubuntu2004-rocm52-py38-torch1110:latest\" - this one already has the nightly installed.",
        "reactions": {
            "url": "https://api.github.com/repos/microsoft/onnxruntime/issues/comments/1192849939/reactions",
            "total_count": 0,
            "+1": 0,
            "-1": 0,
            "laugh": 0,
            "hooray": 0,
            "confused": 0,
            "heart": 0,
            "rocket": 0,
            "eyes": 0
        },
        "performed_via_github_app": null
    },
    {
        "url": "https://api.github.com/repos/microsoft/onnxruntime/issues/comments/1192859870",
        "html_url": "https://github.com/microsoft/onnxruntime/issues/12290#issuecomment-1192859870",
        "issue_url": "https://api.github.com/repos/microsoft/onnxruntime/issues/12290",
        "id": 1192859870,
        "node_id": "IC_kwDOCVq1mM5HGZje",
        "user": {
            "login": "rui-ren",
            "id": 15321482,
            "node_id": "MDQ6VXNlcjE1MzIxNDgy",
            "avatar_url": "https://avatars.githubusercontent.com/u/15321482?v=4",
            "gravatar_id": "",
            "url": "https://api.github.com/users/rui-ren",
            "html_url": "https://github.com/rui-ren",
            "followers_url": "https://api.github.com/users/rui-ren/followers",
            "following_url": "https://api.github.com/users/rui-ren/following{/other_user}",
            "gists_url": "https://api.github.com/users/rui-ren/gists{/gist_id}",
            "starred_url": "https://api.github.com/users/rui-ren/starred{/owner}{/repo}",
            "subscriptions_url": "https://api.github.com/users/rui-ren/subscriptions",
            "organizations_url": "https://api.github.com/users/rui-ren/orgs",
            "repos_url": "https://api.github.com/users/rui-ren/repos",
            "events_url": "https://api.github.com/users/rui-ren/events{/privacy}",
            "received_events_url": "https://api.github.com/users/rui-ren/received_events",
            "type": "User",
            "site_admin": false
        },
        "created_at": "2022-07-22T19:06:57Z",
        "updated_at": "2022-07-22T19:06:57Z",
        "author_association": "MEMBER",
        "body": "Hi @ytaous I upgraded the numpy version and got the same error.\r\n\r\nI am using the platform one, with `pytorch==1.11.0 and rocm52`. \r\n\r\nBelow I think image is released with onnxruntime==1.11.1 \r\n`ptebic.azurecr.io/internal/azureml/aifx/stable-ubuntu2004-rocm52-py38-torch1110:latest` \r\n\r\nHowever, yesterday we released the onnxruntime==1.12.0, if that caused this failure?\r\n\r\nThank you very much for your response.",
        "reactions": {
            "url": "https://api.github.com/repos/microsoft/onnxruntime/issues/comments/1192859870/reactions",
            "total_count": 0,
            "+1": 0,
            "-1": 0,
            "laugh": 0,
            "hooray": 0,
            "confused": 0,
            "heart": 0,
            "rocket": 0,
            "eyes": 0
        },
        "performed_via_github_app": null
    },
    {
        "url": "https://api.github.com/repos/microsoft/onnxruntime/issues/comments/1192862419",
        "html_url": "https://github.com/microsoft/onnxruntime/issues/12290#issuecomment-1192862419",
        "issue_url": "https://api.github.com/repos/microsoft/onnxruntime/issues/12290",
        "id": 1192862419,
        "node_id": "IC_kwDOCVq1mM5HGaLT",
        "user": {
            "login": "ytaous",
            "id": 4484531,
            "node_id": "MDQ6VXNlcjQ0ODQ1MzE=",
            "avatar_url": "https://avatars.githubusercontent.com/u/4484531?v=4",
            "gravatar_id": "",
            "url": "https://api.github.com/users/ytaous",
            "html_url": "https://github.com/ytaous",
            "followers_url": "https://api.github.com/users/ytaous/followers",
            "following_url": "https://api.github.com/users/ytaous/following{/other_user}",
            "gists_url": "https://api.github.com/users/ytaous/gists{/gist_id}",
            "starred_url": "https://api.github.com/users/ytaous/starred{/owner}{/repo}",
            "subscriptions_url": "https://api.github.com/users/ytaous/subscriptions",
            "organizations_url": "https://api.github.com/users/ytaous/orgs",
            "repos_url": "https://api.github.com/users/ytaous/repos",
            "events_url": "https://api.github.com/users/ytaous/events{/privacy}",
            "received_events_url": "https://api.github.com/users/ytaous/received_events",
            "type": "User",
            "site_admin": false
        },
        "created_at": "2022-07-22T19:11:07Z",
        "updated_at": "2022-07-22T19:11:07Z",
        "author_association": "MEMBER",
        "body": "you can uninstall 1.11.1 and try again with the two steps above, do u still have the same error?",
        "reactions": {
            "url": "https://api.github.com/repos/microsoft/onnxruntime/issues/comments/1192862419/reactions",
            "total_count": 0,
            "+1": 0,
            "-1": 0,
            "laugh": 0,
            "hooray": 0,
            "confused": 0,
            "heart": 0,
            "rocket": 0,
            "eyes": 0
        },
        "performed_via_github_app": null
    },
    {
        "url": "https://api.github.com/repos/microsoft/onnxruntime/issues/comments/1193024834",
        "html_url": "https://github.com/microsoft/onnxruntime/issues/12290#issuecomment-1193024834",
        "issue_url": "https://api.github.com/repos/microsoft/onnxruntime/issues/12290",
        "id": 1193024834,
        "node_id": "IC_kwDOCVq1mM5HHB1C",
        "user": {
            "login": "ytaous",
            "id": 4484531,
            "node_id": "MDQ6VXNlcjQ0ODQ1MzE=",
            "avatar_url": "https://avatars.githubusercontent.com/u/4484531?v=4",
            "gravatar_id": "",
            "url": "https://api.github.com/users/ytaous",
            "html_url": "https://github.com/ytaous",
            "followers_url": "https://api.github.com/users/ytaous/followers",
            "following_url": "https://api.github.com/users/ytaous/following{/other_user}",
            "gists_url": "https://api.github.com/users/ytaous/gists{/gist_id}",
            "starred_url": "https://api.github.com/users/ytaous/starred{/owner}{/repo}",
            "subscriptions_url": "https://api.github.com/users/ytaous/subscriptions",
            "organizations_url": "https://api.github.com/users/ytaous/orgs",
            "repos_url": "https://api.github.com/users/ytaous/repos",
            "events_url": "https://api.github.com/users/ytaous/events{/privacy}",
            "received_events_url": "https://api.github.com/users/ytaous/received_events",
            "type": "User",
            "site_admin": false
        },
        "created_at": "2022-07-23T00:50:20Z",
        "updated_at": "2022-07-23T00:50:20Z",
        "author_association": "MEMBER",
        "body": "This issue was from the latest version of onnxruntime-training:\r\n\r\npython -m pip install --pre onnxruntime-training== -f https://download.onnxruntime.ai/onnxruntime_nightly_rocm52.html\r\nLooking in links: https://download.onnxruntime.ai/onnxruntime_nightly_rocm52.html\r\nERROR: Could not find a version that satisfies the requirement onnxruntime-training== (from versions: 1.8.0, 1.8.1, 1.9.0, 1.11.0, 1.11.1, 1.12.0.dev20220706003+rocm52, 1.12.0.dev20220707005+rocm52, 1.12.0.dev20220710001+rocm52, 1.12.0.dev20220711001+rocm52, 1.12.0.dev20220712001+rocm52, 1.12.0.dev20220713001+rocm52, 1.12.0.dev20220714001+rocm52, 1.12.0.dev20220715001+rocm52, 1.12.0.dev20220715002+rocm52, 1.12.0.dev20220718002+rocm52, 1.12.0.dev20220719002+rocm52, 1.12.0.dev20220720003+rocm52, 1.12.0.dev20220721001+rocm52, 1.12.0)\r\nERROR: No matching distribution found for onnxruntime-training==\r\n\r\npip seems to think 1.12.0 is a later version compared to 1.12.0.dev20220721 and pip downloaded the 1.12.0 version from PyPI. which is not a rocm package (it is a cuda package 1.12.0+cu10.2).\r\n\r\nThis issue will be solved by updating the master branch with VERSION_NUMBER == 1.13.0,  and this PR above will fix this issue.\r\n\r\n\r\n",
        "reactions": {
            "url": "https://api.github.com/repos/microsoft/onnxruntime/issues/comments/1193024834/reactions",
            "total_count": 0,
            "+1": 0,
            "-1": 0,
            "laugh": 0,
            "hooray": 0,
            "confused": 0,
            "heart": 0,
            "rocket": 0,
            "eyes": 0
        },
        "performed_via_github_app": null
    }
]